File: app/src/test/java/it/unical/mat/embasp/specializations/solver_planning_domains/PickUp.java
/**
 *
 */
package it.unical.mat.embasp.specializations.solver_planning_domains;

import it.unical.mat.embasp.languages.Id;
import it.unical.mat.embasp.languages.Param;

@Id("pick-up")
public class PickUp {

	@Param(0)
	private String block;

	/**
	 *
	 */
	public PickUp() {
	}

	/**
	 * @return the block
	 */
	public String getBlock() {
		return block;
	}

	/**
	 * @param block
	 *            the block to set
	 */
	public void setBlock(final String block) {
		this.block = block;
	}

}

File: CSharp version/it/unical/mat/test/specialization/solver_planning_domains/PickUp.cs
using it.unical.mat.embasp.languages;

namespace it.unical.mat.test
{
    [Id("pick-up")]
    public class PickUp
    {
        [Param(0)]
        private string block;

        public PickUp()
        {

        }

        public string getBlock()
        {
            return block;
        }

        public void setBlock(string block)
        {
            this.block = block;
        }
    }
}
